### PR TITLE
Set UIDS Base as default theme for HR

### LINKED ISSUE #1606: Convert HR's Article Featured Image to Responsive View Mode
## Problem

HR's article featured image stuff is split and is still using the sitenow 16:9 image

## Proposed Solution
Switch the article full to use the same image settings as default sitenow.

If sitenow 16:9 gets deprecated then this will be mandatory.

### LINKED ISSUE #2325: Add global rule for text area links to break
Add a CSS rule to uids_base for links contained in text areas:
```css
word-break: break-word;
```

This might be able to be applied more broadly, but we would have to think through potential ways this could break existing links.

### LINKED ISSUE #3707: Flip HR to UIDS_Base
Continue work in #3709

## TODO

- [x] Communicate with HR about what work is going to be done before, in dev and after go-live.
- [x] Remove footer sidebar menu blocks
- [x] Place Unit Contact Information Block
- [x] Style Unit Contact Information Block
- [x] Style FAQs similar to uids_base accordions
- [x] Update hook to fix section containers in the absence of a sidebar?

Keep review window short.

## HR Action Items (After Deployment)

- [ ] Find home for Related Content
- [ ] Convert FAQs to Accordions
- [ ] Find spot in main menu for Administrative Services tree.
- [ ] Swap icons in cards

## Once done and workflow agreement with HR is agreed upon.

- [x] Sync HR to DEV/Push Feature Branch

### PR DESCRIPTION
**Pull request by pyrello**
Resolves #1606 
Resolves #3707 
Resolves #2325

# How to test

Check site on hr.dev.drupal.uiowa.edu

## User stories
* As an HR site visitor, I see the UIDS Base theme when I visit the site, which is free of obvious visual deficiencies.
* As an HR webmaster, I can set featured image size defaults for the article content type.

### CODE REVIEW

**Comment by mjoneill** [comment]
Are we waiting for HR to get in touch with us, or are we scheduling a conversation with them?

**Comment by joewhitsitt** [comment]
Meeting scheduled for this Friday.

**Comment by joewhitsitt** [comment]
I rebuilt DEV a couple of days ago and this was lost. Need to fix the merge conflict

**Comment by dnkemper** [comment]
@bspeare I fixed the headline classes and now it's looking sharper IMO. What do you think?
Screen Shot 2021-08-18 at 9 35 07 AM

**Comment by bspeare** [comment]
@dnkemper Looks good.   Can we change the icon title to say just "Contact" and add "Compensation and Classification" as an `<h3>` within the text area?

**Comment by dnkemper** [comment]
@bspeare sure thing! Thank you so much!

File: docroot/sites/hr.uiowa.edu/modules/hr_core/hr_core.install [hunk 1]
```diff
@@ -356,3 +356,36 @@ function hr_core_update_8007() {
...
+    'block.block.hr_content',
+    'block.block.hr_messages',
+    'block.block.hr_local_actions',
+    'block.block.hr_page_title',
+    'block.block.hr_primary_local_tasks',
+    'block.block.hr_secondary_local_tasks',
+    'hr.settings',
+    'uiowa_bar.settings',
+    'uiowa_bootstrap.settings',
+    'uiowa_footer.settings',
```
Comment (richardbporter) [inline docroot/sites/hr.uiowa.edu/modules/hr_core/hr_core.install:381]: Boomshakalaka!

**Comment by bspeare** [comment]
Can we delete the `hr` theme as part of this? It is messing up all the `v2` styles.

**Comment by dnkemper** [comment]
YES!

**Comment by richardbporter** [comment]
> Can we delete the `hr` theme as part of this? It is messing up all the `v2` styles.

Like remove the directory from the codebase? I believe that will bork the uninstall process so it needs to be done in two phases - uninstall and remove the code later.

This is happening even when the HR theme is uninstalled?

**Comment by bspeare** [comment]
Oh, maybe https://github.com/uiowa/uiowa/blob/5bab1d7271001d886a9efd47e9a5a9f0bf652202/docroot/sites/hr.uiowa.edu/modules/hr_core/templates/block--views-block--units-block-contact-info.html.twig#L42 is loading the bootstrap styles in for the contact blocks. :)

**Comment by richardbporter** [comment]
> Oh, maybe
> 
> https://github.com/uiowa/uiowa/blob/5bab1d7271001d886a9efd47e9a5a9f0bf652202/docroot/sites/hr.uiowa.edu/modules/hr_core/templates/block--views-block--units-block-contact-info.html.twig#L42
> 
> is loading the bootstrap styles in for the contact blocks. :)

Good catch. I wouldn't think any theme files would be run if its not installed.

**Comment by joewhitsitt** [comment]
Based on https://github.com/uiowa/uiowa/issues/3707#issuecomment-902272844, removed the WIP label. Leaving as Draft as we don't want to merge this quite yet if approved. Waiting on HR to respond to the workflow plan

**Comment by joewhitsitt** [comment]
Pulling to test

**Comment by joewhitsitt** [comment]
I think this is worth having HR review. Nice work all. Going to put this in DEV and send a email for them to take a look.

**Comment by joewhitsitt** [comment]
Cleared to deploy this Sept 22 (Wednesday) assuming it is still in good shape

**Comment by joewhitsitt** [comment]
Manual changes:

- [ ] Update footer contact block (fixes icons/spacing)
- [ ] Change nav to horizontal
- [ ] Let HR know that Administrative Services needs to be an absolute URL in the footer menu when the main menu item is enabled.

<details>
<summary>footer markup</summary>

```
<p><strong>University Services Building (USB)</strong><br />
1 W. Prentiss Street<br />
Iowa City, Iowa 52240</p>

<p><strong>Benefits:</strong>&nbsp;<a href="tel:319-335-2676"><span class="fa-phone fas"></span>319-335-2676</a><br />
<strong>Payroll:</strong>&nbsp;<a href="tel:319-335-2381"><span class="fa-phone fas"></span>319-335-2381</a><br />
<strong>Administrative Services:</strong>&nbsp;<a href="tel:319-335-3558"><span class="fa-phone fas"></span>319-335-3558</a></p>

<p><a data-entity-substitution="canonical" data-entity-type="node" data-entity-uuid="7c42f330-c0af-4929-99c2-a9ea62ab65dd" href="/node/5206">Accessibility notice and website disclaimers</a></p>
```
</details>

**Review by richardbporter — APPROVED** [review]

**Comment by richardbporter** [comment]
I was just checking what the hook_module_implements_alter() in hr_core was for but couldn't determine that. I thought it was related uiowa_bar/search somehow.

**Comment by joewhitsitt** [comment]
i believe that implements alter is to make sure the form_alter in hr_core fires before sitenow?

**Comment by richardbporter** [comment]
Yea I think my memory is playing tricks on me. PROCEED 👍🏻

**Comment by joewhitsitt** [comment]
https://github.com/uiowa/uiowa/commit/0ee792b1f84207347deabca5a221c388b360d80a

**Pull request merged**